superset/views/sql_lab/views.py

```diff
@@ -31,7 +31,7 @@
 from superset.utils import core as utils
 from superset.utils.core import get_user_id
 
-from .base import BaseSupersetView, DeleteMixin, json_success, SupersetModelView
+from ..base import BaseSupersetView, DeleteMixin, json_success, SupersetModelView
```

Comment:
Could we use the absolute import instead of the relative import?
```python
from superset.views.base import BaseSupersetView, DeleteMixin, json_success, SupersetModelView
```

Reply:
Sure